docs/user_guide/HOWTO/Manage_TPM.rst

```diff
@@ -362,4 +445,25 @@ Trusted Boot Debugging Tips and Warnings
    Be careful updating the ``kernel`` packages and rebuilding the ``initramfs``
    (or running ``dracut``).
 
+HIRS
+----
+
+The SIMP hirs_provisioner module installs and configures the :term:`HIRS` TPM
+Provisoner on specified systems. An Attestation Certificate Authority (ACA)
+must be set up independently. Details of how to do this are provided on the
+:term:`HIRS` website. Additionally, the Acceptance tests in the SIMP 
+hirs_provisioner module include an example of how to do so.
```

Comment:
Do we want to provide a URL for this?

Reply:
The HIRS website is linked in the glossary.

Reply:
I meant a link the the acceptance test

Reply:
Linked to both.